src/librustc/ty/sty.rs

```diff
@@ -2299,23 +2299,33 @@ impl<'tcx> Const<'tcx> {
         assert_eq!(self.ty, ty);
         // if `ty` does not depend on generic parameters, use an empty param_env
         let size = tcx.layout_of(param_env.with_reveal_all().and(ty)).ok()?.size;
+        self.eval(tcx, param_env).val.try_to_bits(size)
+    }
+
+    #[inline]
+    pub fn eval(
```

Comment:
```suggestion
    pub fn interp(
```
(in a follow up)

Reply:
I'm not sure if that is an improvement. We could call it `normalize`?

Reply:
My point was that we use "interp" in other contexts (just look at the module imports 😄) and we have moved away from "eval" so we should be consistent. `normalize` does seem more specific in this context however so that's better than both `interp` and `eval`.